src/fluidsynth.mk

```diff
@@ -0,0 +1,25 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := fluidsynth
+$(PKG)_WEBSITE  := http://fluidsynth.org/
+$(PKG)_DESCR    := FluidSynth - a free software synthesizer based on the SoundFont 2 specifications
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 2.0.5
+$(PKG)_CHECKSUM := 69b244512883491e7e66b4d0151c61a0d6d867d4d2828c732563be0f78abcc51
+$(PKG)_GH_CONF  := FluidSynth/fluidsynth/tags,v
+$(PKG)_DEPS     := cc glib
+
+define $(PKG)_BUILD
+    cd '$(BUILD_DIR)' && '$(TARGET)-cmake' '$(SOURCE_DIR)' \
+        -DBUILD_SHARED_LIBS=$(CMAKE_SHARED_BOOL) \
+	-DCMAKE_BUILD_TYPE=Release
```

Comment:
These two lines aren't necessary, BUILD_SHARED_LIBS and CMAKE_BUILD_TYPE are set by the toolchain file.